### PR TITLE
Implement image uploader functionality

### PR DESCRIPTION
**Pull request by Blager89**

### CODE REVIEW

File: server/routes/upload.js [hunk 1]
```diff
@@ -0,0 +1,31 @@
+const Router = require('koa-router');
+const { authenticated } = require('./../middlewares');
+const { User } = require('./../models');
+const koaBody = require('koa-body');
+const { uploader, validate } = require('./../services');
+
+const router = new Router({prefix:'/api/upload'});
+const allowExtensions = ['image/jpeg', 'image/jpg', 'image/png', 'image/jpg' ];
```
Comment (YuliaFursovych) [inline server/routes/upload.js]: 'image/jpg' -  should be written twice?

File: server/.gitignore [hunk 2]
```diff
@@ -7,6 +7,12 @@ yarn-error.log*
...
 selenium-debug.log
+/static/uploads/avatars/*.jpeg
+/static/uploads/avatars/*.png
+/static/uploads/avatars/*.jpg
+/static/uploads/galleries/*.jpeg
+/static/uploads/galleries/*.png
+/static/uploads/galleries/*.jpg
 
 # Editor directories and files
 .idea
```
Comment (yodeco) [inline server/.gitignore:20]: Optional, add gif file format support

**Review by yodeco — APPROVED** [review]

File: server/routes/upload.js (hunk 1 above)
Comment (Blager89) [reply server/routes/upload.js]: corrected

File: server/.gitignore (hunk 2 above)
Comment (Blager89) [reply server/.gitignore:20]: corrected, but uploader doesn't support gif extensions, if we want upload gif, need add gif extension in validator

**Review by YuliaFursovych — APPROVED** [review]

**Pull request merged**